samples/first_response_file_upload.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python

```

Comment:
This file has Windows newline characters at the end of all of its lines. Please convert to Linux format newlines with only \n

Reply:
fixed.